apps/vmq_server/priv/static/js/status.js

```diff
@@ -109,6 +109,14 @@ $(function() {
         }
     }
 
+    function flatten_minus_one(val) {
```

Comment:
We might just use cap_zero then but why are we "off by one" here?

Reply:
It is a timing issue. There is a short period of time when the total queues is smaller than the nr. of active mqtt connections. This can be seen when a lot of clients subscribe at once to an otherwise "clean" broker. The offline client then switches between -1 and 0 depending on the timing (and stays constantly at zero once all clients connected). I did not cap it at zero, because a -2..-n would point to a different problem. 

https://github.com/vernemq/vernemq/blob/7eab48515756114a573c3f6f621de7ba2e900318/apps/vmq_server/src/vmq_status_http.erl#LL85C9-L85C9

Reply:
Heh, yes, a concurrency thing. The queue setup takes some time and is triggered by the connecting client session.
The hawk eyes you have!